specification/billing/resource-manager/Microsoft.Billing/stable/2024-04-01/billingSavingsPlan.json

```diff
@@ -0,0 +1,1239 @@
+      "type": "object",
+      "properties": {
+        "grain": {
+          "description": "The grain of the aggregate",
+          "readOnly": true,
+          "type": "number"
+        },
+        "grainUnit": {
+          "description": "The grain unit of the aggregate",
+          "readOnly": true,
+          "type": "string"
+        },
+        "value": {
+          "description": "The aggregate value",
+          "readOnly": true,
+          "type": "number"
+        },
+        "valueUnit": {
+          "description": "The aggregate value unit",
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "ExtendedStatusInfo": {
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "statusCode": {
+          "type": "string",
+          "description": "Status code providing additional information."
+        },
+        "message": {
+          "type": "string",
+          "description": "The message giving detailed information about the status code."
+        },
+        "properties": {
+          "type": "object",
+          "description": "Properties specific to credit line check failure",
+          "properties": {
+            "subscriptionId": {
+              "type": "string",
+              "description": "The subscription that has failed credit line check."
+            }
+          }
+        }
+      }
+    },
+    "SavingsPlanUpdateValidateRequest": {
+      "type": "object",
+      "properties": {
+        "benefits": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SavingsPlanUpdateRequestProperties"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "SavingsPlanValidateResponse": {
+      "type": "object",
+      "properties": {
+        "benefits": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SavingsPlanValidResponseProperty"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Url to get the next page."
+        }
+      }
+    },
+    "SavingsPlanValidResponseProperty": {
+      "type": "object",
+      "description": "Benefit scope response property",
+      "properties": {
+        "valid": {
+          "type": "boolean",
+          "description": "Indicates if the provided input is valid"
+        },
+        "reasonCode": {
+          "type": "string",
+          "description": "Failure reason code if the provided input is invalid"
+        },
+        "reason": {
+          "type": "string",
+          "description": "Failure reason if the provided input is invalid"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "ExpandParameter": {
+      "name": "expand",
+      "description": "May be used to expand the planInformation.",
+      "in": "query",
+      "x-ms-parameter-location": "client",
```

Comment:
Please confirm if this parameter is client level. Since it is not used in all operations, it should be method level.